Flutter/pdf-viewer/form-filling.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: post
+title: Form filling in Flutter PDF Viewer widget | Syncfusion
+description: Learn here all about form filling feature of Syncfusion Flutter PDF Viewer (SfPdfViewer) widget and more.
+platform: Flutter
+control: SfPdfViewer
+documentation: ug
+---
+
+# Form filling in Flutter PDF Viewer (SfPdfViewer)
+
+The [SfPdfViewer](https://pub.dev/documentation/syncfusion_flutter_pdfviewer/latest/pdfviewer/SfPdfViewer-class.html) allows you to fill, edit, save, export, and import the AcroForm fields in a PDF document.
+
+## Supported form fields
+
+You can load and fill in the following form fields in a PDF document using the Flutter PDF Viewer. 
+
+* Text box. 
+* Checkbox. 
+* Radio button. 
+* Combo box. 
+* Signature. 
+
+## Save form data 
+
+You can save the modified form fields data by calling the `saveDocument` method. Refer to the following code example. 
```

Comment:
You can save the modified form field data by calling the `saveDocument` method. Refer to the following code example.